lib/bond/generated/CsProtocol_writers.hpp

```diff
@@ -233,15 +226,15 @@ void Serialize(TWriter& writer, ::CsProtocol::Device const& value, bool isBase)
     } else {
         writer.WriteFieldOmitted(BT_STRING, 9, nullptr);
     }
-#ifdef HAVE_CS4
+
```

Comment:
Is it intended to make CS4 as default serialization?

Reply:
No. I ran the bond compiler to regenerate the classes to add the new fields, and it overwrote this file. I didn't realize we were manually modifying a file that is supposed to only be generated by the bond compiler. this will need to be fixed.